test/Sentry.Tests/Internals/ILSpy/SingleFileAppTests.cs

```diff
@@ -30,7 +30,9 @@ static SingleFileAppTests()
         InValidBundleFile = Path.Combine(testRoot, invalidBundle);
     }
 
-#if NET8_0
+#if NET9_0
+    private static string TargetFramework => "net9.0";
```

Comment:
A revew for line 45. Instead of `throw` which will cause it to compile and fail at runtime, use:

```
#error "Target Framework not yet supported for single file apps"
```